design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,50 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
```

Comment:
Since task is usually nested under a story, do we need both links or just the task is sufficient?

Reply:
It makes sense in ADO but I have seen instances in github where tasks don't have a story mentioned as parent. This will help keep track of the same.